Gems/PhysX/Code/Source/EditorRigidBodyComponent.h

```diff
@@ -92,6 +92,8 @@ namespace PhysX
         void BuildGameEntity(AZ::Entity* gameEntity) override;
 
         const AzPhysics::RigidBody* GetRigidBody() const;
+        const bool IsSceneCCDDisabled() const;
+        void OpenPhysXConfigurationPane() const;
```

Comment:
update: `const bool` -> `bool` and also should `OpenPhysXConfigurationPane` be `private`? Does anything else need this outside the class?

Reply:
OpenPhysXConfigurationPane is defined in EditorRigidBodyComponent and is being accessed by EditorRigidBodyConfiguration, I could move it there but it doesn't really make sense there